api/admin/controller.py

```diff
@@ -168,6 +170,24 @@ def details(self, data_source, identifier):
         return entry_response(
             AcquisitionFeed.single_entry(self._db, work, annotator)
         )
+        
+    def complaints(self, data_source, identifier):
+        """Return detailed complaint information for admins."""
+        
+        pool = self.load_licensepool(data_source, identifier)
```

Comment:
If you're starting with a LicensePool I don't know if it makes sense to consolidate complaints for all the LicensePools associated with that work. 

Complaints are registered with the LicensePool. Some can only be fixed by fixing the Work ("wrong audience"), some by fixing the Edition associated with the work ("wrong title") and some by fixing the LicensePool itself ("book won't render"). Right now there's generally one Edition and one LicensePool associated with each book, so treating all those errors the same way is okay for now... I guess I'm just worried about the future here.